### PR TITLE
Balanced portfolio

### PR DESCRIPTION
**Pull request by metdinov**
Solves #34, #31

### CODE REVIEW

**Comment by metdinov** [comment]
Added tests #36

**Comment by metdinov** [comment]
Changed to WIP, some design changes need to handle multiple data sources.

File: backtester/datahandler/balanced_datahandler.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+
+class BalancedDataHandler(DataHandler):
+    """Handler for balanced data set"""
+
+    def __init__(self, data_path, events):
+        data = pd.read_csv(data_path, parse_dates=["date"])
+
+        # We will assume bid and ask prices = close
+        data["bid"] = data["close"]
```
Comment (unbalancedparentheses) [inline backtester/datahandler/balanced_datahandler.py]: I think that on another PR I have sen that you were correctly using bid and ask as two separate prices. Let's try to use the same criteria everywhere.

Comment (unbalancedparentheses) [reply backtester/datahandler/balanced_datahandler.py]: Sorry my mistake, we don't have this information.

**Pull request merged**